misk/src/main/kotlin/misk/web/WebConfig.kt

```diff
@@ -93,6 +94,9 @@ data class WebConfig(
   /** The level of log when concurrency shedding. */
   val concurrency_limiter_log_level: Level = Level.ERROR,
```

Comment:
nit: It would be nice to move `concurrency_limiter_disabled` and `concurrency_limiter_log_level` into the new `concurrency_limiter: ConcurrencyLimiterConfig` backwards compatibly for better config organization. This would take a few PRs / rolling the changes out.

Reply:
agreed! how would you recommend we do so? I wanted to touch as little as possible, but i'm happy to add the additional configurations. are you suggesting i'd go through services using this configuration and change them?

Reply:
Yes, I'd land equivalents in your new `ConcurrencyLimiterConfig` and update any callsites to look first at the new field in `ConcurrencyLimiterConfig` and then in the old one if not present. Then, update any services using the old fields to use the new ones. Then, delete the old fields from `WebConfig`.

Reply:
any concerns about Misk being _open source_?